conftest.py

```diff
@@ -140,9 +140,9 @@ def _allclose(a, b, rtol=1e-5, atol=1e-8, xtol=0,
             indices along the first axis
         equal_nan : bool
             If True, nans will be considered equal to nans.
-        print_fail : bool
-            If True, print out the first 5 entries failing the allclose check
-            along the first axis.
+        print_fail : int
+            If > 0, print out the first `print_fail` entries failing
```

Comment:
Earlier in this docstring, it uses `*` to mark a variable: `*xtol*`. Is that an oversight, or should I be using that here, too? I think the two should be consistent.

Reply:
That should be double backticks too.  We could use bold instead of literals for referring to variable names if we wanted, it's just a style choice.  But we've been using the backticks elsewhere in the code from what I can see.